.github/workflows/pr-2-confirm.yml

```diff
@@ -69,6 +69,7 @@ jobs:
           if [[ "${version}" == "null" ]]; then
             echo "before=null" >> $GITHUB_OUTPUT
             echo "after=1.0" >> $GITHUB_OUTPUT
+            exit 0
```

Comment:
Just a note that this is an important addition that short-circuits the rest of the logic that otherwise errors without a valid `${version}`.